AcsEmulator/AcsEmulatorAPI/Endpoints/CallAutomation/CallAutomationController.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using AcsEmulatorAPI.Models;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
That's where the IsNullOrEmpty collection extension method comes from, we should probably replace that ... later.